packages/core/services/slack/constructPresentAndSquadJoinMessage.ts

```diff
@@ -0,0 +1,67 @@
+import type { ChatPostMessageArguments } from "@slack/web-api";
+
+import {
+  additionalPresentIdeasInputActionId,
+  additionalPresentIdeasInputBlockId,
+  additionalPresentIdeasSaveButtonActionId,
+  squadJoinCheckboxActionId,
+  squadJoinCheckboxBlockId,
+} from "@/types/SlackInteractionRequest";
+
+import {
+  makeActionsBlock,
+  makeCheckboxInputBlock,
+  makeInputBlock,
+  makeTextBlock,
+} from "./messageItems";
+
+type Arguments = {
+  birthdayPerson: string;
+  user: string;
+  name: string;
+  eventId?: string;
+};
+
+export const constructPresentAndSquadJoinMessage = ({
+  birthdayPerson,
+  user,
+  name,
+  eventId,
+}: Arguments): ChatPostMessageArguments =>
+  ({
+    channel: user,
+    metadata: eventId
+      ? {
+          event_type: "askPresentIdeas",
+          event_payload: {
+            eventId,
+          },
+        }
+      : undefined,
+    text: `Hey, <@${user}>! <@${birthdayPerson}>'s birthday is in 2 months. Do you have any additional present ideas?`,
```

Comment:
The is in 2 months is a bit weird because we said that in the previous message and now its 4 days later.

Maybe less than 2? Or getting closer?